docs/install.rst

```diff
@@ -70,122 +97,150 @@ by executing the following command in your conda env:
-```radical-entk-version``` command-line tool. The current version should be
-printed.
+        You can build and execute the container by running:
 
-.. code-block:: bash
+        .. code-block:: bash
 
-        radical-entk-version
-        0.70.0
+                docker build -f ./Dockerfile -t entk .
+                docker run -t -i entk
 
+        Once you execute the container, the default path will be /root (of the container).
+        The EnTK virtualenv exists at ~/ve-entk (inside the container). This is useful
+        to know as the examples exist inside the virtualenv.
 
-Installing rabbitmq
-===================
+        You can check the version of Ensemble Toolkit with the
+        ```radical-entk-version``` command-line tool. The current version should be
+        printed.
 
-Installing rabbitmq as a system process (sudo privileges required)
-------------------------------------------------------------------
+        .. code-block:: bash
 
-Ensemble Toolkit relies on RabbitMQ for message transfers. Installation
-instructions can be found at <https://www.rabbitmq.com/download.html>. At
-the end of the installation run ```rabbitmq-server``` to start the server.
-RabbitMQ needs to be installed on the same machine as EnTK is installed.
+                radical-entk-version
+                0.70.0
 
-In some cases, you might have to explicitly start the rabbitmq-server after
-installation. You can check if the rabbitmq-server process is alive. If not,
-please run the following:
+
+RabbitMQ
+========
+
+Ensemble Toolkit relies on RabbitMQ for message transfers. RabbitMQ needs to be
+configured or it can be installed on the same machine as EnTK is installed.
+Installation instructions can be found at
+<https://www.rabbitmq.com/download.html>. At the end of the installation run
+```rabbitmq-server``` to start the server.
+
+The following configuration defines a default server and port number to communicate.
 
 .. code-block:: bash
 
-        rabbitmq-server -detached
+        export RMQ_HOSTNAME=two.radical-project.org; export RMQ_PORT=33239
 
+.. comments
 
-Installing rabbitmq using docker
---------------------------------
+        Installing rabbitmq
+        ===================
 
-If installing rabbitmq directly seems to be cumbersome, you can also install a
-docker instance of rabbitmq. Assuming you have docker installed, you can
-download and run the rabbitmq instance using the following command:
+        Installing rabbitmq as a system process (sudo privileges required)
+        ------------------------------------------------------------------
 
-.. code-block:: bash
+        Ensemble Toolkit relies on RabbitMQ for message transfers. Installation
+        instructions can be found at <https://www.rabbitmq.com/download.html>. At
+        the end of the installation run ```rabbitmq-server``` to start the server.
+        RabbitMQ needs to be installed on the same machine as EnTK is installed.
```

Comment:
I would eliminate `RabbitMQ needs to be installed on the same machine as EnTK is installed.` as it is not generally true.